_Week8/Group3/simulate.do

```diff
@@ -0,0 +1,54 @@
+* first, drop any program called 'schools'
+cap prog drop schools
+
+* Now we create a program 'schools' that will store results in r()
+prog def schools, rclass
+	syntax, rho(real) // input is intracluster correlation coefficient ('ICC')
```

Comment:
I am not sure that `rho` here is exactly equivalent to the resulting ICC -- it certainly is not in your output! Perhaps it is equivalent to the _expected_ ICC in a large sample?